tools/ios/util.bzl

```diff
@@ -57,6 +57,34 @@ def ios_pipeline(
   needsXCTest = False,
   bundle_name = None
 ):
+
+  native.sh_test(
+    name = name + "SwiftLint",
+    srcs = [":"+ name + "_Lint"],
+    visibility = ["//visibility:public"],
+  )
+
+  native.genrule(
+    name = name + "_Lint",
+    tools = [
+      "@SwiftLint//:swiftlint"
+    ],
+    srcs = native.glob(["Sources/**/*.swift"]) + ["//:.swiftlint.yml"],
+    outs = ["output.sh"],
+    executable = True,
+    visibility = ["//visibility:public"],
+    cmd="""
+      echo `$(location @SwiftLint//:swiftlint) --config $(location //:.swiftlint.yml) $(SRCS) || true` > lint_results.txt
+      LINT=$$(cat lint_results.txt)
+
+      echo '#!/bin/bash' > $(location output.sh)
+      echo "echo '$$LINT'" > $(location output.sh)
+
+      LINESWITHERROR=$$(echo grep error lint_results.txt || true)
+      echo "exit $$(($$LINESWITHERROR) | wc -l)" >> $(location output.sh)
+  """
+  )
+  
```

Comment:
can you move the lint definitions to the bottom of the macro?

The doc block has to be at the top of the function to meet the [docstring format](https://skydoc.bazel.build/docs/writing.html#macro-documentation)